CMakeLists.txt

```diff
@@ -1573,32 +1576,30 @@ else()
 endif()
 add_dependencies(mixxx-lib mixxx-res mixxx-script)
 
-file(READ src/_version.h MIXXX_VERSION_FILECONTENT)
-string(REGEX REPLACE "^.*#define MIXXX_VERSION \"(.*)\".*$" "\\1" MIXXX_VERSION "${MIXXX_VERSION_FILECONTENT}")
-# a dummy configure to force a reconfigure when the version changes
-configure_file(src/_version.h src/_version.h)
+if (MIXXX_VERSION_PRERELEASE STREQUAL "")
+   set(MIXXX_VERSION "${CMAKE_PROJECT_VERSION}")
+else()
+   set(MIXXX_VERSION "${CMAKE_PROJECT_VERSION}-${MIXXX_VERSION_PRERELEASE}")
+endif()
+configure_file(src/_version.h.in src/_version.h @ONLY)
```

Comment:
```suggestion
configure_file(src/version.h.in src/version.h @ONLY)
```

Reply:
The issue is that we have already version.h in utils a second one will be confusing. 
How about renaming the one in utils?

Reply:
I have now rename _version.h to version.h and Version class to VersionStore. 
Locks nice and clean now. 
What do you think?